workspaces/libnpmexec/lib/index.js

```diff
@@ -40,6 +40,8 @@ const exec = async (opts) => {
     yes = undefined,
     ...flatOptions
   } = opts
+  delete flatOptions.packageLock
+  delete flatOptions.packageLockOnly
```

Comment:
We shouldn't `delete` things from `flatOptions` since we pass it everywhere and it could have unintended side effects. I think creating a new object and omitting these specifically is a better option.

Reply:
This should happen in the command itself too.